### DISCUSSION TITLE
Unable to withdraw funds after channels are closed

### DISCUSSION THREAD

**Discussion by fishbanana21** [lightningnetwork/lnd#7956] [Troubleshooting]
Hello!

I have had my node for about a year. I have had around 30 channels. 20 of which got closed around 2-3 months ago if I remember correctly, and the last 1/3 about a week ago when I decided to close all channels, as the plan is to start fresh with a new install. With just bitcoin core and lnd, with no overhead UI like I have now. 

However when I went to withdraw funds I realized that all funds show as pending onchain in the Node UI (Citadel)

I found this strange especially for the sats belonging to the channels closed months ago, not a single satoshi shows as available on chain. 

I ran lncli listunspent and here I see a lot of unspent balances. So I wonder what would be the correct way to get them out / get them spendable? 

No channels are active now.

**Comment by guggero** [comment]
Can you check if your node is fully synced? And some additional info would be nice, for example the output of `lncli listunspent` and `lncli pendingchannels`.

**Reply by fishbanana21** [reply]
Currently show block : 806 214

lncli pendingchannels
{
    "total_limbo_balance": "0",
    "pending_open_channels": [
    ],
    "pending_closing_channels": [
    ],
    "pending_force_closing_channels": [
    ],
    "waiting_close_channels": [
    ]
}

and I will add a textfile with `lncli listunspent` as that one is alot of output from.

**Reply by guggero** [reply]
Okay, I have a suspicion what that could've been caused by.
Can you restart `lnd` with `reset-wallet-transactions=true` in your config file and then let it re-scan (this can take a very long time, hours on some weaker hardware) and then see if that resolves itself.
See ["Forced in-place rescan"](https://github.com/lightningnetwork/lnd/blob/master/docs/recovery.md#forced-in-place-rescan) doc for more info.

**Reply by fishbanana21** [reply]
That seemed to fix it! Thank you so much! I realised after posting by searching some more that I found the problem mentioned earlier. And fixed in PR 7800. 

I sent you some sats as a little thank you for your fast and awesome help :)

**Comment by fishbanana21** [comment]
See comment from @guggero 

Okay, I have a suspicion what that could've been caused by.
Can you restart lnd with reset-wallet-transactions=true in your config file and then let it re-scan (this can take a very long time, hours on some weaker hardware) and then see if that resolves itself.
See ["Forced in-place rescan"](https://github.com/lightningnetwork/lnd/blob/master/docs/recovery.md#forced-in-place-rescan) doc for more info.
